src/Components/List/ListEmpty.tsx

```diff
@@ -0,0 +1,54 @@
+// Libraries
+import React, {forwardRef} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {StandardFunctionProps, ComponentSize} from '../../Types'
+
+export interface ListEmptyProps extends StandardFunctionProps {
+  /** Controls whether the text contents of this item wrap or not */
+  wrapText?: boolean
+  /** Size of this component */
+  size?: ComponentSize
+}
+
+export type ListEmptyRef = HTMLDivElement
+
+export const ListEmpty = forwardRef<ListEmptyRef, ListEmptyProps>(
```

Comment:
I wonder if we should call this `ListEmptyState` in keeping with then `EmptyState` component use.